CONTRIBUTING.rst

```diff
@@ -67,11 +67,10 @@ With respect to code standards, we follow `PEP8`_ and the `Google Style Guide`_.
 .. _`Google Style Guide` : https://google.github.io/styleguide/pyguide.html
 
 
-Licensing
+Licensing of contributions
 --------
 
-The License of this project is located in `LICENSE`_. By submitting a contribution to this project, you are agreeing that your contributions under this license and
-with this waiver of copyright interest.
+By submitting a contribution to this project, you are granting the project maintainers the ability to release your contribution under the project license located in `LICENSE`_. Along with this, you also waiver copyright interest.
```

Comment:
@jackiekazil I actually think the original language is more correct, since (as I understand it) the copyright waiver is part of the Apache license itself; this wording makes it sound like the copyright waiver is a separate thing.

@rht that's an interesting question. I think American courts at least are inconsistent on that (e.g. software EULAs that you click 'agree' to are binding, but 'by using this website you agree to ...' type things are not). I don't know what the rules are with regard to open-source contributions.